### PR TITLE
HPCC-12951 Refresh WU Lists on Tab Selection

### PR DESCRIPTION
**Pull request by GordonSmith**
Fixes HPCC-12951

Signed-off-by: Gordon Smith gordon.smith@lexisnexis.com

### CODE REVIEW

**Comment by GordonSmith** [comment]
@miguelvazq Please Review

File: esp/src/eclwatch/GraphWidget.js [hunk 1]
```diff
@@ -1033,40 +1033,19 @@ define([
...
-                        return origFunc(node, addStyle, removeStyle);
+                ESPUtil.MonitorVisibility(this, function (visibility, node) {
+                    if (visibility) {
+                        dojo.style(node, "width", "100%");
+                        dojo.style(node, "height", "100%");
+                        dojo.style(node.firstChild, "width", "100%");
+                        dojo.style(node.firstChild, "height", "100%");
+                        return true;
+                    } else {
+                        dojo.style(node, "width", "1px");
```
Comment (miguelvazq) [inline esp/src/eclwatch/GraphWidget.js]: Wouldn't visibility:hidden be a more effective of hiding something?

Comment (GordonSmith) [inline esp/src/eclwatch/GraphWidget.js]: This is specifically for the ActiveX control and hiding it causes it to be destroyed in Chrome (which is why this code is here in the first place)!

**Comment by miguelvazq** [comment]
Then looks good.

**Pull request merged**